recipe/meta.yaml

```diff
@@ -181,6 +180,157 @@ outputs:
+        - if not exist %LIBRARY_BIN%\crt\link.stub exit 1  # [win]
+    about:
+      home: https://developer.nvidia.com/cuda-toolkit
+      license_file: LICENSE
+      license: LicenseRef-NVIDIA-End-User-License-Agreement
+      license_url: https://docs.nvidia.com/cuda/eula/index.html
+      summary: CUDA internal tools.
+      description: |
+        CUDA internal tools.
+      doc_url: https://docs.nvidia.com/cuda/index.html
+
+  - name: cuda-crt-dev_{{ target_platform }}
+    build:
+      noarch: generic
+    run_exports:
+      strong:
+        - cuda-version >={{ cuda_version }},<{{ cuda_version_next_major }}
+    files:
+      - targets/{{ target_name }}/include/crt  # [linux]
+      - Library\include\crt                    # [win]
+    requirements:
+      host:
+        - arm-variant * {{ arm_variant_type }}  # [aarch64]
+        - cuda-version {{ cuda_version }}
+      run:
+        - {{ pin_compatible("cuda-version", max_pin="x.x") }}
+      run_constrained:
+        - arm-variant * {{ arm_variant_type }}    # [aarch64]
+    test:
+      commands:
+        - test -f $PREFIX/targets/{{ target_name }}/include/crt/common_functions.h  # [linux]
+        - if not exist %LIBRARY_INC%\crt\common_functions.h exit 1  # [win]
+    about:
+      home: https://developer.nvidia.com/cuda-toolkit
+      license_file: LICENSE
+      license: LicenseRef-NVIDIA-End-User-License-Agreement
+      license_url: https://docs.nvidia.com/cuda/eula/index.html
+      summary: CUDA internal headers.
+      description: |
+        CUDA internal headers.
+      doc_url: https://docs.nvidia.com/cuda/index.html
+
+  - name: cuda-nvvm-tools
+    files:
+      - nvvm/bin                # [linux]
+      - nvvm/libdevice          # [linux]
+      - Library\nvvm\bin        # [win]
+      - Library\nvvm\libdevice  # [win]
+    requirements:
+      build:
+        - {{ compiler("c") }}
+        - {{ compiler("cxx") }}
+      host:
+        - arm-variant * {{ arm_variant_type }}  # [aarch64]
+        - cuda-version {{ cuda_version }}
+      run:
+        - {{ pin_compatible("cuda-version", max_pin="x.x") }}
+      run_constrained:
+        - arm-variant * {{ arm_variant_type }}    # [aarch64]
+    test:
+      commands:
+        - test -d $PREFIX/nvvm                                    # [linux]
+        - test -f $PREFIX/nvvm/bin/cicc                           # [linux]
+        - if not exist %LIBRARY_PREFIX%\nvvm\bin\cicc.exe exit 1  # [win]
+        - if not exist %LIBRARY_PREFIX%\nvvm\libdevice exit 1     # [win]
+    about:
+      home: https://developer.nvidia.com/cuda-toolkit
+      license_file: LICENSE
+      license: LicenseRef-NVIDIA-End-User-License-Agreement
+      license_url: https://docs.nvidia.com/cuda/eula/index.html
+      summary: Compiler for CUDA applications.
+      description: |
+        Compiler for CUDA applications.
+      doc_url: https://docs.nvidia.com/cuda/index.html
+
+  - name: cuda-nvvm-dev_{{ target_platform }}
```

Comment:
This needs to be `noarch: generic` just like `cuda-crt-dev_{{ target_platform }}` is

It is needed because `cuda-nvcc-dev_{{ target_platform }}` is `noarch: generic` and depends on both of these

Without `noarch: generic` the compiler becomes uninstallable in cross-compilation workloads

```suggestion
  - name: cuda-nvvm-dev_{{ target_platform }}
    build:
      noarch: generic
```

Reply:
Fixing in PR ( https://github.com/conda-forge/cuda-nvcc-impl-feedstock/pull/12 ), which has more context